static/app/gettingStartedDocs/node/node.tsx

```diff
@@ -22,18 +22,66 @@ import {
 
 type Params = DocsParams;
 
-const getSdkSetupSnippet = () => `
+const getExampleServerSnippet = () => `
 ${getImportInstrumentSnippet()}
-
-// All other imports below
 const { createServer } = require("node:http");
 
-const server = createServer((req, res) => {
-  // server code
+const server = createServer(async (req, res) => {
+    res.statusCode = 200;
+    res.setHeader('Content-Type', 'text/plain');
+    res.end('Hello World');
 });
 
-server.listen(3000, "127.0.0.1");
-`;
+server.listen(3000, "127.0.0.1", () => {
+    console.log(\`Server running at http://127.0.0.1:3000/\`);
+  });`;
+
+const getErrorsVerifySnippet = () => `
+// Capture an exception
+try {
+  foo();
+} catch (e) {
+  Sentry.captureException(e);
+}`;
+
+const getPerformanceVerifySnippet = () => `
+const Sentry = require("@sentry/node");
+
+// Start a span
+Sentry.startSpan({
+  op: "test",
+  name: "My First Test Span",
+}, async () => {
+  ${getErrorsVerifySnippet()}
+});`;
+
+const hasContinuousProfiling = (params: DocsParams) =>
+  params.isProfilingSelected &&
+  params.profilingOptions?.defaultProfilingMode === 'continuous';
+
+const getContinuousProfilingVerifySnippet = () => `
+    async function slow_function() {
+        Sentry.profiler.startProfiler();
+        await new Promise(resolve => setTimeout(resolve, 5000));
+        Sentry.profiler.stopProfiler();
+    }
+    // Start a span
+    await Sentry.startSpan({
```

Comment:
Won't this complain that await is only valid in async functions? Or is the validate more for illustration purposes?